db/migrations/20220609044825-move-github-app-id-to-subscriptions.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.removeColumn("Installations", "githubAppId");
+
+		await queryInterface.addColumn("Subscriptions", "gitHubAppId", {
+			type: Sequelize.INTEGER,
+			allowNull: true
+		});
+	},
+
+	down: async (queryInterface, Sequelize) => {
+		await queryInterface.addColumn("Installations", "gitHubAppId", {
```

Comment:
Should this be back to `githubAppId` instead of  `gitHubAppId`?

Reply:
No. Changed it to `gitHubAppId` because we're trying to standardise the way we use the word GitHub in the app. The 'H' should be captialised and should have been that way when we first introduced this field.

Reply:
I agree with the case change in general, I was more meaning the specific `down` function for `Installations` since it probably has to go back to the dumb spelling it was, if it does the down-grade? It might not care ¯\_(ツ)_/¯